docs/architecture.md

```diff
@@ -174,8 +174,8 @@ Every operation that enters the exchange pipeline will receive a result, either
 immediately (read: synchronously) or eventually as the result of a network
 request comes in.
 
-The raw GraphQL result from an API is typically: `{ data?: T, errors?: GraphQLError[] }`.
-And `urql`'s operation results are very similar: `{ data?: T, error?: CombinedError }`.
+The raw GraphQL result from an API is typically: `{ data?: T, errors?: GraphQLError[], extensions?: Record<string, any> }`.
```

Comment:
Small nit, we can remove "typically" from this sentence because I was referring to extensions as an exception from the typical results